src/ServiceControl.Persistence.RavenDb5/ServiceControl.Persistence.RavenDb5.csproj

```diff
@@ -6,12 +6,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\ServiceControl.Infrastructure\ServiceControl.Infrastructure.csproj" />
-    <ProjectReference Include="..\ServiceControl.Persistence\ServiceControl.Persistence.csproj" />
+    <!-- Private=false & ExcludeAssets=runtime prevent repeatedly including binary dependencies of ServiceControl.Persistence in each persister directory -->
```

Comment:
Makes sense as those assemblies are already there at runtime. I thought copying was only done for packages and executables but likely that is pre SDK behavior.